Update troubleshooting regarding ASCII user names (#3647)
EnterpriseDB/docs: advocacy_docs/supported-open-source/postgresql/installer/04_installation_troubleshooting.mdx

  - "/edb-docs/d/postgresql/installation-getting-started/installation-guide-installers/10/PostgreSQL_Installation_Guide.1.22.html"
---

## Multi-byte characters in user name

> If your system user name contains non-ASCII characters, the installation may fail. Make user your user name or machine name does not include multi-byte characters.

Reviewer: ```suggestion
If your system user name contains non-ASCII characters, the installation may fail. Make sure your user name or machine name does not include multi-byte characters.
```

> If your system user name contains non-ASCII characters, the installation may fail. Make sure your user name or machine name does not include multi-byte characters.

Reviewer: ```suggestion
If your system user name or machine name contains non-ASCII characters, the installation may fail. Make sure your user name or machine name don't include multi-byte characters.
```
